src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountSteps.java

```diff
@@ -0,0 +1,127 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageobjects.Accounts.SFADetailsPage;
+import org.fundacionjala.sfdc.pageobjects.Accounts.SFAMainPage;
+import org.fundacionjala.sfdc.pageobjects.Accounts.SFANewModifyPage;
+import org.fundacionjala.sfdc.pageobjects.SalesForceMainTabClassic;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * AccountSteps.java
+ * Class for steps of account.feature.
+ */
+public class AccountSteps {
+    private WebDriver webDriver;
+    private Properties propertiesManager;
+
+    /**
+     * Given step.
+     */
+    @Given("^Go to accounts section$")
+    public void goToAccountsSection() {
+        webDriver = DriverManager.getInstance().getNavigator();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)